src/simple_cache_client.rs

```diff
@@ -2312,6 +2256,36 @@ impl SimpleCacheClient {
         Ok(MomentoDeleteResponse::new())
     }
 
+    /// Generates a token for Momento
```

Comment:
Consistency in verbiage will help with the clarity of these functions: I see you seem to use "token," "api token" and "api key" interchangeably (I think?) in this pr. We've also got a "session token" for sso so this isn't getting any clearer!

Reply:
yep thats correct

Reply:
resolved in https://github.com/momentohq/client-sdk-rust/pull/132/commits/a10baf2df4d874788a9eae80e49a54fbedf939bd